jina/drivers/evaluate.py

```diff
@@ -5,14 +5,15 @@
 
 import numpy
 
-from . import BaseExecutableDriver, RecursiveMixin
+from . import BaseExecutableDriver
 from ..types.querylang.queryset.dunderkey import dunder_get
 from .search import KVSearchDriver
 from ..types.document import Document
 from ..types.document.helper import DocGroundtruthPair
+from ..types.sets.doc_groundtruth import DocumentGroundtruthSequence
 
 
-class BaseEvaluateDriver(RecursiveMixin, BaseExecutableDriver):
+class BaseEvaluateDriver(BaseExecutableDriver):
```

Comment:
I think this should also inherit `ContextAwareRecursiveDriver` by having `DocGroundTruthPair` implement ` traversal`

Reply:
The point is, that the `__call__` function itself is overwriten. So nothing from the `ContextAwareRecursiveMixin` remains. For the sake of "that's what I want to do here as a higher concept" we could keep the mixin, but it would not change anything.

Reply:
then remove okey!